docs/src/content/docs/id/guides/css-and-tailwind.mdx

```diff
@@ -22,7 +26,7 @@ Sesuaikan style yang diterapkan pada website Starlight Anda dengan memberikan fi
 
 2. Tambahkan path file CSS Anda ke dalam array `customCss` pada config Starlight di `astro.config.mjs`.
 
-   ```js
+   ```diff lang="js"
```

Comment:
It looks like this is missing the [`+` diff indicators on 2 lines of code](https://github.com/withastro/starlight/pull/1102/files#diff-6429e29a42a27836094af0ffa6527c90616b826a953a84fa2e615ffd04cf64e5R35-R36) in this code block.

Reply:
Ah i see, thank you for notice this. I will fix this soon.